apps/nextjs-website/src/messages/it.json

```
@@ -100,14 +100,16 @@
         "relatedLinksTitle": "Link utili",
         "speakersTitle": "Ospiti",
         "questionList": {
-            "createdAt": "Ora creazione",
-            "text": "Domanda",
+            "title": {
+                "highlightedQuestions": "Domande evidenziate",
+                "notHighlightedQuestions": "Domande"
```

```suggestion
                "questions": "Domande"
```
why not just questions?